src/Datadog.Trace/TracingProcessManager.cs

```diff
@@ -230,6 +236,20 @@ private static bool ProgramIsRunning(string fullPath)
             return false;
         }
 
+        private static string GetProcessIdFromFileName(string fullPath)
+        {
+            var fileName = Path.GetFileNameWithoutExtension(fullPath);
+
+            if (fileName == null)
+            {
+                return "-1";
```

Comment:
This is basically a throwaway value that will indicate a process isn't alive, happy to change this or add proper null checks.

Reply:
Returning null does seem more idiomatic.